engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -287,7 +287,7 @@ public void copyInitialState(Path origAppDir) throws IOException
     FileStatus[] lFiles = fs.listStatus(origAppDir);
     for (FileStatus f : lFiles) {
       if (f.isDirectory()) {
-        String targetPath = f.getPath().toString().replace(origAppDir.toString(), newAppDir);
```

Comment:
A comment showing a sample of what this path string looks like as well as origAppDir and newAppDir would be helpful for readers and future maintainers to understand the subtleties here.